src/panel/poi/PoiPanel.jsx

```diff
@@ -268,28 +253,21 @@ export default class PoiPanel extends React.Component {
         text: _('Back to list'),
         className: 'poi_panel__back_to_list',
```

Comment:
Nit: There is only one version left of this, so this could be put inline in the `Flex` element.